arm:
  joint_states_controller:
      type: joint_state_controller/JointStateController
      publish_rate: 30

  arm_zero_joint_effort_controller:
    type: effort_controllers/JointEffortController
    joint: arm_zero_to_child_neck
    pid: {p: 1000, i: 10, d: 0.01 }

  arm_one_joint_effort_controller:
    type: effort_controllers/JointEffortController
    joint: arm_one_to_arm_zero
    pid: {p: 1000, i: 10, d: 0.01 }

  arm_two_joint_effort_controller:
    type: effort_controllers/JointEffortController
    joint: arm_two_to_arm_one
    pid: {p: 1000, i: 10, d: 0.01 }

  arm_three_joint_effort_controller:
    type: effort_controllers/JointEffortController
    joint: arm_three_to_arm_two
    pid: {p: 1000, i: 10, d: 0.01 }

  arm_four_joint_effort_controller:
    type: effort_controllers/JointEffortController
    joint: arm_four_to_arm_three
    pid: {p: 1000, i: 10, d: 0.01 }

  hand_joint_effort_controller:
    type: effort_controllers/JointEffortController
    joint: hand_to_arm_four
    pid: {p: 1000, i: 10, d: 0.1 }
